app/services/manual/withdraw_service.rb

```diff
@@ -40,8 +40,6 @@ def remove_from_search_index
 
   def manual
     @manual ||= Manual.find(manual_id, user)
-  rescue KeyError => error
-    raise ManualNotFoundError.new(error)
```

Comment:
I think this should be dealt with separately, because `ManualWithdrawer#execute` is rescuing `Manual::WithdrawService::ManualNotFoundError`. I think the latter should probably be rescuing `Manual::NotFoundError` instead and that change would probably be better done at the same time as this.